generate.php

```diff
@@ -6,11 +6,12 @@
 // ------------------- //
 const VERSION = 0.1;
 const TYPE_GIT = 'git';
-$tempFolder = './tmp';
+$tempFolder = './third-party/';
 
 $extensions = [];
 $gitRepositories = [];
 if (file_exists($tempFolder)) {
+	// TODO: Improve by keeping git copy if possible (e.g. fetch + reset)
 	exec("rm -rf -- {$tempFolder}");
```

Comment:
Help welcome for some commands to keep the local git copies and update them via git safely